include/functions/utility.hpp

```diff
@@ -8,6 +8,8 @@
 #define BIG_INT_UTILITY_FUNCTIONS_HPP
 
 #include <tuple>
+#include <stdint.h>
```

Comment:
Please change this to `<cstdint>`.

Reply:
Correct me if i'm wrong, would that require every reference to uint64_t to be prefixed with std::?

Reply:
Because in other c++ code that I've seen they don't use the std:: prefix

Reply:
No, you wouldn't need the `std::` namespace for `int64_t` and `uint64_t`